.github/workflows/ci-external.yaml

```diff
@@ -0,0 +1,21 @@
+name: CI External
+
+on:
+  workflow_dispatch:  # Allow manual triggers
+  pull_request:
+  push:
```

Comment:
I don't think we need to trigger it on `push`. Actually we had that in the past and we decided to remove it because it overlapped with pull_request. The latter already allows the CI to run with external contributions

Reply:
Okay, just removed this one with external CI.